completers/ufw_completer/cmd/actions/profile.go

```diff
@@ -0,0 +1,21 @@
+package actions
+
+import (
+	"github.com/carapace-sh/carapace"
+	"strings"
+)
+
+func ActionUfwProfiles() carapace.Action {
+	return carapace.ActionExecCommand("sudo", "ufw", "app", "list")(func(output []byte) carapace.Action {
```

Comment:
So far I'm avoiding `sudo` for security (and some other) reasons.
Completion might also be invoked as `root` with `sudo` missing which wouldn't work in this case.

```suggestion
	return carapace.ActionExecCommand("ufw", "app", "list")(func(output []byte) carapace.Action {
```

I'll give privilege elevation some thought in the future.

Reply:
Yeah, I was thinking the same about `sudo`. This should work well too.